samples/Config.CustomConfigurationProviders.Sample/Config.CustomConfigurationProviders.Sample.csproj

```diff
@@ -4,13 +4,14 @@
 
   <PropertyGroup>
     <IsPackable>false</IsPackable>
-    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <TargetFrameworks>netcoreapp2.0;net461</TargetFrameworks>
     <OutputType>Exe</OutputType>
   </PropertyGroup>
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Extensions.Configuration" Version="$(AspNetCoreVersion)" />
     <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="$(AspNetCoreVersion)" />
+    <PackageReference Include="System.Collections.Immutable" Version="$(ImmutableCollectionsVersion)" />
```

Comment:
Doesn't seem to break if included for `netcoreapp2.0`.  Also, was not conditionally included before the app was converted: https://github.com/aspnet/Entropy/commit/7f3238253b2085444ae521ca77fbc24378a39217#diff-ce7d231799e41a8d9205c984b36c85b0.  I can file an issue to change this if we think it's incorrect.